openy_calc/src/Form/CalcBlockForm.php

```diff
@@ -127,42 +127,56 @@ public function buildForm(array $form, FormStateInterface $form_state) {
       '#markup' => $header,
     ];
 
+    $form['membership_calc_body'] = [
+      '#type' => 'container',
+      '#attributes' => [
+        'class' => ['membership-calc__body'],
+      ],
+    ];
+
     switch ($step) {
       case 1:
         // Membership type step.
-        $form['type'] = [
+        $form['membership_calc_body']['type'] = [
           '#element_variables' => $types,
           '#subtype' => 'membership_type_radio',
           '#type' => 'calc_radios',
-          '#title' => $this->t('Which option best describes the type of membership you need?'),
+          '#title' =>  '<h3>' . $this->t('Choose your membership type') . '</h3>',
           '#options' => $types_options,
           '#default_value' => $storage['type'] ?? NULL,
         ];
         break;
 
       case 2:
         // Select branch step.
-        $form['map'] = [
-          '#type' => 'openy_map',
-          '#element_variables' => $this->dataWrapper->getBranchPins(),
-        ];
         $locations = $this->dataWrapper->getLocations();
         $locations_options = [];
+        $locations_options[0] = $this->t('Choose location');
         foreach ($locations as $id => $location) {
           $locations_options[$id] = $location['title'];
         }
```

Comment:
**Optional change:**
Single responsibility, so could be changed to something like
$locations_options = [0 => $this->t('Choose location')] + array_map(function ($location) {
  return $location['title'];
}, $locations);

Reply:
Fixed